docs/reference/collection-configuration.txt

```diff
@@ -10,51 +10,88 @@ Collection Configuration
    :depth: 2
    :class: singlecol
 
+Configuring a Document Collection
+=================================
 
-Capped Collections
-------------------
+You can specify collection options for documents using ``store_in`` macro.
+This macro accepts ``:collection_options`` argument, which can contain any collection
+options that is supported by the driver.
+
+.. note::
+
+  In order to apply the options, the collection must be explicitly created up-front.
+  This should be done using  :ref:`Collection Management Rake Task<collection-management-task>`.
+
+Please refer to `the driver collections page
+<https://mongodb.com/docs/ruby-driver/current/reference/collection-tasks/>`_
+for the more information about collection options.
+
+.. note::
 
-Mongoid does not provide a mechanism for creating capped collections on the fly - you
-will need to create these yourself one time up front either with the driver or via the
-Mongo console.
+  Collection options depend on the driver version and MongoDB server version.
+  It is possible that some options, like time series collectios, are not available
+  on older server versions. Mongoid does not validate collection options.
```

Comment:
The "Mongoid does not validate" bit could potentially be interpreted as that users are on their own for validation, but this isn't actually the case - either the driver or the server will validate the options. I suggest either adding something like "but you may receive an error from the driver or from the server for using options that are not supported",  replacing the last sentence ("Mongoid does not validate") with this bit or removing the "Mongoid does not validate" bit without replacement.

Reply:
Done